src/sql/ResourceSelector.test.tsx

```diff
@@ -1,7 +1,8 @@
-import React from 'react';
 import { render, screen } from '@testing-library/react';
-import { ResourceSelector, ResourceSelectorProps } from './ResourceSelector';
+import React from 'react';
```

Comment:
I'm used to having React as first, any reason why you moved it?

Reply:
I'm now using [Typescript Hero](https://marketplace.visualstudio.com/items?itemName=rbbit.typescript-hero) to automatically add, order and remove imports. Since it's automatic, it uses the import path to order them (alphabetically). It also differentiate between packages and relative imports.

Reply:
I see! 
I would have thought it would put default before named imports, but I don't see anything in the eslint or prettier config for it so good to go 👍